coda/coda_mdstore/tests/test_views.py

```diff
@@ -591,22 +591,42 @@ def test_raises_http404_file_handle_is_falsy(self, rf):
         with pytest.raises(http.Http404):
             views.bagURLList(request, bag.name)
 
-    @pytest.mark.xfail(reason='Refactor is required.')
-    def test_output_text(self):
-        assert 0
-
-    @pytest.mark.xfail(reason='Refactor is required.')
-    def test_top_files_block(self):
-        # TODO: rename once implemented.
-        assert 0
-
-    @pytest.mark.xfail(reason='Refactor is required.')
-    def test_path_is_not_unicode_safe(self):
-        assert 0
+    def test_output_text(self, rf):
+        """Test the outputText obtained from mocked getFileHandle object"""
```

Comment:
I think this docstring doesn't really clarify anything. There's no need to test the output from a mocked object, as we control exactly what a mocked object returns.

Reply:
I think I can rephrase it as `Test the outputText obtained contains mocked getFileHandle return_value`, 'cos this test checks if the mocked object's return_value gets processed correctly and goes to `outputText` variable. What do you think?